packages/core/src/components/Grid/Col.tsx

```diff
@@ -47,7 +47,7 @@ export default withStyles(({ unit }) => {
     };
 
     offsets[`offset${offset}`] = {
-      marginLeft: offset > 0 ? 100 / (12 / offset) : 0,
+      marginLeft: offset > 0 ? `${100 / (12 / offset)}%` : 0,
```

Comment:
Was this actually breaking before?

Reply:
yep! didn't notice it in the examples because of the default `space-between`

Reply:
<img width="1013" alt="Screen Shot 2019-04-19 at 6 25 22 PM" src="https://user-images.githubusercontent.com/306275/56449598-88d6aa00-62d0-11e9-9295-b7fbb720cce7.png">

Reply:
Nice catch :P

Reply:
😎